projects/VitalRecord/IJE.cs

```diff
@@ -607,29 +609,37 @@ protected string PresenceToIJE(bool fieldValue, bool noneOfTheAboveValue)
             {
                 return "N";
             }
-            else
+            else if (this.Record.IsCategoryEmpty(fhirFieldName))
```

Comment:
Could you add some comments that explain our approach where U is only returned if the category is completely empty and if a U was provided in IJE with other Ys and/or Ns in the category it won't round trip?